ios/RCTWebRTC/RTCVideoViewManager.m

```diff
@@ -263,11 +267,41 @@ - (void)setVideoTrack:(RTCVideoTrack *)videoTrack {
     // of its videoTrack only while this view resides in a window.
     if (videoTrack && self.window) {
         dispatch_async(_module.workerQueue, ^{
-            [videoTrack addRenderer:self.videoView];
+            [videoTrack addRenderer:self];
         });
     }
   }
 }
+#pragma mark - RTCVideoRenderer methods
+
+/**
+ * Renders a specific video frame. Delegates to the subview of this instance
+ * which implements the actual {@link RTCVideoRenderer}.
+ *
+ * @param frame The video frame to render.
+ */
+- (void)renderFrame:(RTCVideoFrame *)frame {
+    id<RTCVideoRenderer> videoRenderer = self.videoView;
+    if (!firstFrameRendered) {
```

Comment:
In case if I do not provide callback, the App crushes, throwing `EXC_BAD_ACCESS` in Xcode.

I'm not an expert in Objective-C, but this seems to be enough as a quick fix:
```suggestion
    if (self.onFirstFrame && !firstFrameRendered) {
```

Also, note, that I haven't tested/checked Android version.